apps/api/src/domain/models/vocabulary-list/commands/import-entries-to-vocabulary-list/import-entries-to-vocabulary-list.command.integration.spec.ts

```diff
@@ -0,0 +1,129 @@
+                    person: '11',
+                    positive: true,
+                },
+            },
+        ],
+        variables: [],
+    }
+);
+
+const dummyFsa = buildTestCommandFsaMap().get(
+    commandType
+) as CommandFSA<ImportEntriesToVocabularyList>;
+
+const aggregateCompositeIdentifier = {
+    type: AggregateType.vocabularyList,
+    id: termId,
+};
+
+const vocabularyListId = dummyFsa.payload.aggregateCompositeIdentifier.id;
+
+const commandFsaFactory = new DummyCommandFsaFactory<ImportEntriesToVocabularyList>(() =>
+    clonePlainObjectWithOverrides(dummyFsa, {
+        payload: {
+            aggregateCompositeIdentifier,
+            entries: [
+                {
+                    termId,
+                    propertyValues: {},
+                },
+            ],
+        },
+    })
+);
+
+describe(commandType, () => {
+    let app: INestApplication;
+
+    let databaseProvider: ArangoDatabaseProvider;
+
+    let testRepositoryProvider: TestRepositoryProvider;
+
+    let commandHandlerService: CommandHandlerService;
+
+    let idManager: IIdManager;
+
+    let commandAssertionDependencies: CommandAssertionDependencies;
+
+    beforeAll(async () => {
+        ({ testRepositoryProvider, commandHandlerService, idManager, app, databaseProvider } =
+            await setUpIntegrationTest({
+                ARANGO_DB_NAME: generateDatabaseNameForTestSuite(),
+            }).catch((error) => {
+                throw error;
+            }));
+
+        commandAssertionDependencies = {
+            testRepositoryProvider,
+            idManager,
+            commandHandlerService,
+        };
+    });
+
+    beforeEach(async () => {
+        await testRepositoryProvider.testSetup();
+    });
+
+    afterAll(async () => {
+        await app.close();
+
+        databaseProvider.close();
+    });
+
+    describe(`when the command is valid`, () => {
+        it(`should succeed`, async () => {
+            await assertCommandSuccess(commandAssertionDependencies, {
+                systemUserId: dummySystemUserId,
+                buildValidCommandFSA: () =>
+                    commandFsaFactory.build(undefined, {
+                        aggregateCompositeIdentifier: {
+                            id: vocabularyListId,
+                        },
+                        entries: {
+                            termId,
+                        },
+                    }),
+                seedInitialState: async () => {
+                    await testRepositoryProvider
+                        .forResource(ResourceType.vocabularyList)
+                        .create(existingVocabularyList);
+                },
+                checkStateOnSuccess: async () => {
+                    const vocabularyListSearchResult = await testRepositoryProvider
+                        .getCategoryRepository()
+                        .fetchById(vocabularyListId);
+                },
+            });
+        });
+    });
+});
```

Comment:
Invalid cases:
- one of the properties in `propertyValues` is not registereD
- one of the property values is not allowed
- one of the terms does not exist